JsonSchema/SchemaKeywordRegistry.cs

```diff
@@ -102,6 +102,7 @@ static SchemaKeywordRegistry()
 		};
 
 		_keywords = new ConcurrentDictionary<string, Type>(keywordData.ToDictionary(x => x.Item2, x => x.Item1));
+		_keywordTypeInfoResolvers = new ConcurrentDictionary<Type, JsonSerializerContext>(keywordData.ToDictionary(x => x.Item1, _ => (JsonSerializerContext)JsonSchemaSerializerContext.Default));
 
```

Comment:
RuleRegistry did this differently and just returns null for known types. It seems more efficient to not populate this dictionary and just return the default context when the keyword type isn't found.